includes/helper/views/html-licences.php

```diff
@@ -12,11 +12,46 @@
 <h1 class="screen-reader-text"><?php esc_html_e( 'Licenses', 'wp-job-manager' ); ?></h1>
 <div class="wpjm-licences">
 	<?php if ( ! empty( $licenced_plugins ) ) : ?>
+		<?php if ( ! empty( $show_bulk_activate ) ) : ?>
+		<div class="wpjm-bulk-activate">
+			<b class="wpjm-bulk-activate--title">
+				Activate Job Manager Licenses
```

Comment:
Missing internationalisation in all these copies.

Reply:
Good catch! Fixed in 154266c97b5afbc23348943a034c7ce72a90839f.